cmd/virtual-workspaces/command/cmd.go

```diff
@@ -60,6 +60,10 @@ func NewCommand(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command {
 		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
 
 		Run: func(c *cobra.Command, args []string) {
+			if err := opts.Logs.ValidateAndApply(); err != nil {
+				klog.Fatal(err)
```

Comment:
this print stack trace. Better print the error normally, and then os.Exit

Reply:
better: use the RunE variant for the command and just return the error, also below

Reply:
good call. I converted to RunE but kept the block that prints details if the error was invalid. Is that still useful, or should I remove it, too?

Reply:
honestly I don't understand why that code is there. Would remove it.

Reply:
Done